### PR TITLE
Update docs for 0.38.0 release

### PR DESCRIPTION
**Pull request by jt-nti**

### CODE REVIEW

File: src/markdown-pages/docs/running-simbank-tests/running-simbank-tests-cli.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ In order to run the Galasa SimBanks tests you need to add some configuration inf
 
 The SimBank tests are held in the <a href="https://github.com/galasa-dev/simplatform" target="_blank"> Galasa simplatform repository</a> in GitHub. To start running the tests you need to clone the repository, if you have not already done so. To find out how to clone the cli repository, follow the instruction in the [Running Galasa SimBank online](../running-simbank-tests/simbank-cli) documentation.
 
-After cloning the repository, complete the following steps to run the SimBankIVT test that is provided with Galasa. The following example uses SimBank OBR version `0.25.0` and Galasa uber OBR version `0.37.0`.
+After cloning the repository, complete the following steps to run the SimBankIVT test that is provided with Galasa. The following example uses SimBank OBR version `0.25.0` and Galasa uber OBR version `0.38.0`.
```
Comment (techcobweb) [inline src/markdown-pages/docs/running-simbank-tests/running-simbank-tests-cli.md:53]: I think the simbank obr was bumped also ? @jadecarino ?

File: src/markdown-pages/highlights.md [hunk 2]
```diff
@@ -18,11 +18,31 @@ We have the following available Slack channels:
...
 
 
-## 0.37.0 - Release Highlights
+## 0.38.0 - Release Highlights
+
+- CLI updates:
+
+  - You can get and delete secrets from the credentials store using the `galasactl secrets get` and `galasactl secrets delete` commands.
+
+    [Managing Ecosystem encryption keys](./docs/ecosystem/ecosystem-managing-encryption-keys.md) describes how to use `galasactl secrets get` to replace the encryption key being used to encrypt credentials in the Galasa Ecosystem's credentials store.
```
Comment (techcobweb) [inline src/markdown-pages/highlights.md:27]: secrets also work with the `galasactl apply -f <filename.yaml>` syntax

Comment (techcobweb) [reply src/markdown-pages/highlights.md:27]: Secrets also support base64 encoding so special characters are never going to be mangled by any codepage conversions...etc.

File: src/markdown-pages/docs/running-simbank-tests/running-simbank-tests-cli.md (hunk 1 above)
Comment (jt-nti) [reply src/markdown-pages/docs/running-simbank-tests/running-simbank-tests-cli.md:53]: Jade updated that in #843 

https://github.com/galasa-dev/galasa.dev/blob/next/src/markdown-pages/docs/running-simbank-tests/running-simbank-tests-cli.md?plain=1#L53

**Review by eamansour — APPROVED** [review]
Can't spot anything else that hasn't been included or covered under bug fixes and security updates - approved

**Pull request merged**